parser/quakec-parser.js

```diff
@@ -707,6 +721,7 @@ class Parse {
                 {line: -1, character: -1}
             );
 
+            Context.symbols.push(Context.token);
```

Comment:
This pushes the end token onto the symbol stack. It does not appear to have any negative consequences, and fixes the issue with semicolon checks appearing in the wrong spot at EOF.